KeepAwake@jepfa.de/extension.js

```diff
@@ -200,7 +220,7 @@ function updateMode() {
 function showModeTween() {
 
   
-    if (_mode == MODE_ON) {
+    if (_mode == MODE_ON || _mode == MODE_ON_LOCK) {
         _tweenText = new St.Label({ style_class: 'video-label-on', text: _("Computer keeps awake.") });
```

Comment:
I think it is better to show an additional message when toggling to MODE_ON_LOCK, e.g. "Computer keeps awake even after restart. "